src/components/EmbeddedAskForm.scss

```diff
@@ -1,46 +1,43 @@
 @import '../shared/shared';
 
-.form-container {
+.form-container header { 
   // Match .container
   margin: auto;
   width: 92% !important;
   max-width: 850px;
   padding: 1em;
+}
 
-  &.open {
-    .form-toggle-button {
-      background: #d9d9d9; // grey
-    }
-    .embedded-form {
-      transition: max-height 1s;
+.form-container.open {
```

Comment:
I'm going to merge this, but I'd ask to try to use the nesting SASS syntax in the future -- it's generally in use throughout and the `&.class` syntax and nesting helps me clarify the hierarchy.